File: Day-14/Pattern17.py
n = int(input())
def PatternSeventeen(n):
    i = 1
    while(i <=n):
        j = 0
        while ( j <i):
            print(chr(65 + j + i -1), end='')
            j += 1
        i += 1
        print(end='\n')
    
PatternSeventeen(n)

File: Day-15/Pattern17.js
function PatternSeventeen(n) {
  let i = 1;
  while (i <= n) {
    let j = 0;
    let str = "";
    while (j < i) {
      str += String.fromCharCode(65 + j + i - 1);
      j++;
    }
    console.log(str);
    i++;
  }
}

PatternSeventeen(4);
